contracts/axelar-gateway/src/contract.rs

```diff
@@ -173,13 +173,18 @@ impl AxelarGatewayInterface for AxelarGateway {
         env: Env,
         signers: axelar_soroban_interfaces::types::WeightedSigners,
         proof: axelar_soroban_interfaces::types::Proof,
+        enforce_rotation_delay: bool,
     ) {
         let data_hash: BytesN<32> = env
             .crypto()
             .keccak256(&(CommandType::RotateSigners, signers.clone()).to_xdr(&env))
             .into();
 
-        // TODO: Add rotation delay governance
+        if enforce_rotation_delay {
```

Comment:
```suggestion
        if !enforce_rotation_delay {
```
operator is required when delay is not enforced. Double check the tests/behaviour with EVM to help catch this kind of issues